README.rst

```diff
@@ -68,7 +68,7 @@ PUDL is comprised of three core components:
     to the original inputs. Each of the data inputs may have several different versions
     archived, and all are assigned a unique DOI and made available through the REST API.
     You can read more about the Raw Data Archives in the
-    `docs <https://catalystcoop-pudl.readthedocs.io/en/dev/intro.html#raw-data-archives>`__.
+    `docs <https://catalystcoop-pudl.readthedocs.io/en/nightly/#raw-data-archivess>`__.
```

Comment:
These links to `intro.hml` were broken.  And also I updated all the direct docs links to use the newly default `nightly` docs.

Reply:
No, looks like a broken link. Well - it just sends you to the main page without popping you to the anchor.

Reply:
Yesss, my preciousssss.

Reply:
Fixed it locally will push in a sec.